src/Relation/Nullary/Decidable/Core.agda

```diff
@@ -52,6 +51,15 @@ open Dec public
 pattern yes p =  true because ofʸ  p
 pattern no ¬p = false because ofⁿ ¬p
 
+------------------------------------------------------------------------
+-- decision procedures are Decidable
```

Comment:
I feel these should live in `Data.Bool.Properties`?

Reply:
Well I suppose that is part of the question here. My thoughts had been towards moving `Dec`/`Decidable` questions out of `Data.Bool.*` in favour of `Relation.Nullary.Decidable.*` (which is already importing `Bool`...), which further simplifies the imports of `Data.Bool.Properties` (the dependency graph does get a bit hairy back-and-forth between the two, and moreover the `Other properties` section at the end of `Data.Bool.Properties` feels a bit of an orphan, unless it was placeholder for further development?